nixos/modules/services/web-servers/nginx/gitweb.nix

```diff
@@ -0,0 +1,91 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.nginx.gitweb;
+  gitwebConfigFile = pkgs.writeText "gitweb.conf" ''
```

Comment:
Could this not be shared with the lighttpd version? In a project of this size, DRY is very important, so updates apply everywhere…

Reply:
@wmertens Done, thanks!